about.css

```diff
@@ -0,0 +1,326 @@
+  background-size: cover;
+}
+
+.event-p {
+  font-size: 14px;
+  font-weight: 900;
+}
+
+footer {
+  display: flex;
+  background-color: #524f4f;
+  margin-top: 50px;
+}
+
+.footer-img {
+  width: 50%;
+  margin-left: 20px;
+  margin-top: 10px;
+}
+
+.footer-div {
+  display: flex;
+  flex-direction: column;
+}
+
+.closeBtn {
+  margin-left: 90%;
+}
+
+.fab {
+  width: 100%;
+}
+
+.partners {
+  display: none;
+}
+
+.main-logo {
+  display: none;
+}
+
+.head-links {
+  display: none;
+}
+
+.nav-jsi a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.aboutpg {
+  background-image: url(./images/agric\ summit\ 2.webp);
+  background-repeat: no-repeat;
+  width: 100%;
+  background-size: 243%;
+}
+
+@media screen and (min-width: 720px) {
+  .nav-ul {
+    display: flex;
+    gap: 60px;
+    justify-content: flex-end;
+    margin-right: 15%;
+  }
+
+  .nav-li {
+    list-style-type: none;
+  }
+
+  .news {
+    border: 5px #ec5242 solid;
+    padding: 2px 20px;
+    margin-top: -5px;
+  }
+
+  .fa {
+    display: none;
+  }
+
+  .aboutpg {
+    background-image: url(./images/agric\ summit\ 2.webp);
+    background-repeat: no-repeat;
+    width: 100%;
+    background-size: 100%;
+  }
+
+  .gtg {
+    font-family: "Lato", sans-serif;
+    font-size: 35px;
+    text-align: center;
+  }
+
+  .topic {
+    color: #ec5242;
+    font-size: 50px;
+    text-align: center;
+    width: 60%;
+    margin-left: 20%;
+    margin-top: -4px;
+    padding-top: 4%;
+    margin-bottom: 80px;
+  }
+
+  .pg {
+    text-align: center;
+    color: black;
+    border: 3px solid #fff;
+    box-shadow: 0 5px 5px rgb(222 222 222 / 82%), 0 5px 5px rgb(222 222 222 / 82%);
+    padding: 20px 20px;
+    font-family: "Lato", sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    margin-bottom: 40px;
+    margin-left: 27%;
+    width: 40%;
+    background-color: #fff;
+  }
+
+  .details {
+    text-align: center;
+  }
+
+  .d-link {
+    color: #000;
+    font-weight: 900;
+  }
+
+  .agro-theme {
+    text-align: center;
+  }
+
+  .agro-talk {
+    text-align: center;
+  }
+
+  .b-logo {
+    height: auto !important;
+    margin-left: 43%;
+  }
+
+  .past-events {
+    text-align: center;
+  }
+
+  .event-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    gap: 70px;
+  }
+
+  .event-d {
+    color: #fff;
+    position: relative;
+  }
+
+  .event-cover {
+    width: 100%;
+    padding: 4rem 0;
+    background-color: rgba(236, 82, 66, 0.7);
+  }
+
+  .event-1 {
+    background-image: url(./images/agric\ summit.jpg);
+    background-position: center;
+    background-size: cover;
+  }
+
+  .event-2 {
+    background-image: url(./images/agric\ summit\ 4.webp);
+    background-position: center;
+    background-size: cover;
+  }
+
+  .event-p {
+    font-size: 14px;
+    font-weight: 900;
+  }
+
+  footer {
+    display: flex;
+    background-color: #524f4f;
+    margin-top: 50px;
+  }
```

Comment:
- [ ] The design for the footer on the desktop version is different from that of the original design. In the original design, there is a little gap between the logo and the copyright contents while in yours, there is a wide gap. Also, the background colour of the footer on the desktop version is `#272a31` while you are using a `#524f4f`. Please note the text colour used is white. Kindly implement these changes 👍 😎 

### Your Design

![image](https://github.com/Danny-imo/Capstone-project/assets/77106962/53264c4e-850c-4f22-aac5-1bdf2a4d96d4)

### Behance Design

![image](https://github.com/Danny-imo/Capstone-project/assets/77106962/9a0cd7d3-578e-4908-9cf4-c1da077b4dfc)